packages/shared/src/components/sidebar/MyFeedButton.tsx

```diff
@@ -22,10 +23,9 @@ function MyFeedButton({
   alerts,
   onNavTabClick,
 }: MyFeedButtonProps): ReactElement {
+  const { user } = useContext(AuthContext);
   const myFeedMenuItem: SidebarMenuItem = {
-    icon: (active: boolean) => (
-      <ListIcon Icon={() => <HomeIcon secondary={active} />} />
-    ),
+    icon: () => <ProfilePicture user={user} size="xsmall" />,
```

Comment:
Oh, cool. I have actually did some change where it can accept a `ReactNode` since in the implementation it actually checks if the parameter passed is a function. Let's see what can easily work better when merging the conflict.